### PR TITLE
Refactor short summaries

### PR DESCRIPTION
**Pull request by jarrodmillman**
The proposals are due tomorrow.  I think the last issue was to make the two short summaries clearer about how they relate to one another and how they differ.  We want to make sure it is clear that these two talks will have minimal overlap.

Feel free to suggest edits or push directly to my branch.

- [ ] Check you name, affiliations, etc.
- [ ] Add your personal web page if you wish

### CODE REVIEW

**Review by tupui — APPROVED** [review]
LGTM, thanks for the update @jarrodmillman

File: presentations/proposed/scipy2022/maintainers/proposal.md [hunk 1]
```diff
@@ -48,3 +48,28 @@ Track: Maintainers
 Author: Pamphile T. Roy
 Keywords: Scientific Python, Community
 Type:  Talk
+corresponding author
+
+### Author 2
```
Comment (tupui) [inline presentations/proposed/scipy2022/maintainers/proposal.md:53]: Then I would do this as well. Also not sure what this char means `†`.

If you have personal websites, we can also add these.


```suggestion
corresponding author

### Author 1

First name†: Pamphile T.
Last name: Roy
Email: roy.pamphile@gmail.com 	
Country/region: Austria
Organization: Quansight

### Author 2
```

File: presentations/proposed/scipy2022/maintainers/proposal.md [hunk 2]
```diff
@@ -12,9 +12,11 @@ Scientific Python---By Maintainers and for Maintainers
...
 ## Short Summary
 
-The future of Scientific Python is here.
-Behind the scenes, core scientific projects have started coordinating to better unify the community.
-Learn more about the goals of this unification effort, opportunities that exist, and, most importantly, how to join the adventure.
+The Scientific Python project aims to better coordinate the ecosystem and grow the community.
+This talk focuses on our efforts to better coordinate the ecosystem.
+Come hear how behind the scenes, core scientific projects have started coordinating to better unify the community.
+Learn more about the goals of this unification effort, opportunities that
+exist, and, most importantly, how to join the adventure.
```
Comment (stefanv) [inline presentations/proposed/scipy2022/maintainers/proposal.md:19]:
```suggestion
The Scientific Python project aims to better coordinate the ecosystem and grow the community.
This talk focuses on our efforts to better coordinate project development, and to improve shared infrastructure.
Come hear about our goals, our recent technical work, and learn how you can get involved!
```

**Review by stefanv — APPROVED** [review]

File: presentations/proposed/scipy2022/general/proposal.md [hunk 3]
```diff
@@ -54,3 +56,46 @@ Track: General
...
 Keywords: Scientific Python, Community
 Type:  Talk
+corresponding author
+
+### Author 1
+
+First name†: Juanita
+Last name: Gomez
+Email: juanitagomezr2112@gmail.com
+Country/region: USA
```
Comment (juanis2112) [inline presentations/proposed/scipy2022/general/proposal.md:66]:
```suggestion
Country/region: Colombia
```

**Review by juanis2112 — APPROVED** [review]
I changed my country/region to Colombia but otherwise LGTM! Thanks @jarrodmillman

**Pull request merged**